extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -865,6 +865,7 @@ export class AmpA4A extends AMP.BaseElement {
   viewportCallback(inViewport) {
     if (this.friendlyIframeEmbed_) {
       setFriendlyIframeEmbedVisible(this.friendlyIframeEmbed_, inViewport);
+      this.friendlyIframeEmbed_.setCssAnimPlayingState(inViewport);
```

Comment:
Can `friendly-iframe` just observe the `visibilityObservable_` to toggle `'i-amphtml-animations-paused'` class instead of exporting a function that has to be called manually?